rotkehlchen/api/v1/schemas.py

```diff
@@ -205,7 +205,7 @@ def make_ethereum_transaction_query(  # pylint: disable=no-self-use
 
 
 class EthereumTransactionDecodingSchema(AsyncIgnoreCacheQueryArgumentSchema):
-    tx_hashes = fields.List(EVMTransactionHashField(), required=True)
+    tx_hashes = fields.List(EVMTransactionHashField(), load_default=[])
```

Comment:
why not make it `required=False`, which should make it `None` and then check for None?